src/common/MessageDispatcher.sol

```diff
@@ -529,28 +577,21 @@ contract MessageDispatcher is Auth, IMessageDispatcher {
     }
 
     /// @inheritdoc IVaultMessageSender
-    function sendUpdateShares(
-        PoolId poolId,
-        ShareClassId scId,
-        address receiver,
-        D18 pricePoolPerShare,
-        uint128 shares,
-        bool isIssuance
-    ) external auth {
+    function sendUpdateShares(PoolId poolId, ShareClassId scId, uint128 shares, bool isIssuance) external auth {
+        // TODO: Remove price in Hub
+        D18 price = d18(1, 1);
```

Comment:
Note: open TODO here

Reply:
@onnovisser can be fixed with current main